napari/_vispy/visuals/clipping_planes_mixin.py

```diff
@@ -22,10 +22,11 @@ class ClippingPlanesMixin:
     """
 
     def __init__(self: _PVisual, *args, **kwargs) -> None:
-        self._clip_filter = PlanesClipper()
+        clip_filter = PlanesClipper()
         super().__init__(*args, **kwargs)
 
-        self.attach(self._clip_filter)
```

Comment:
Because `self` is typed as `_PVisual`, which doesn't have a `._clip_filter` attribute, `mypy` doesn't like this. The change here makes `mypy` happy for now, but in the future would be good for someone to look into if `self` should really be typed as `_PVIsual` or not.

Reply:
Yeah, I think `self: _PVisual` is wrong. `self` should always be self. Plus `PVisual` is only used here.

+1 to opening an issue once this is merged.

Reply:
I think this is due to a misuse of mixins where we assume that a mixin will be used with a particular type of class. This comes up in other parts of the code too (e.g. `IntensityVisualizationMixin`).

+1 that an issue would be much appreciated!